### DISCUSSION TITLE
connection not possible RING

### DISCUSSION THREAD

**Discussion by bordino** [dgreif/ring#760] [General]
hello, I installed the plugin on hoobs 4.0.100 and I cannot validate my RING username / password in the plugin (chrome,safari or Firefox…), what to do, please?

it is for a video doorbell 4.

Thank you

**Comment by dgreif** [comment]
I recommend you reach out to HOOBS support

**Reply by bordino** [reply]
where can I contact them, please?

**Reply by dgreif** [reply]
https://support.hoobs.org/

**Comment by bordino** [comment]
Configuration

Install the Ring Plugin = **it’s good**

To connect to the Ring Servers you need to login with the ring login credentials on the plugin configurations page = **« IMPOSSIBLE »**

**Comment by bordino** [comment]
Hello @dgreif it's good it works 😃.
for homekit what can I activate, please?
because I am French and with google translate I do not understand all the settings 😅.

**Reply by dgreif** [reply]
I'm not exactly sure what you mean by `for homekit what can I activate, please`.  Could you clarify?

**Reply by bordino** [reply]
hello 
sorry I misspoke.  
Could you explain to me the different settings that are in the plugin because I do not understand them all, please?

**Reply by dgreif** [reply]
They are all detailed here: https://github.com/dgreif/ring/tree/master/homebridge#optional-parameters
